bakery/codec_test.go

```diff
@@ -0,0 +1,61 @@
+package bakery
+
+import (
+	"math/rand"
+
+	gc "gopkg.in/check.v1"
+)
+
+type codecSuite struct{}
+
+var _ = gc.Suite(&codecSuite{})
+
+func (*codecSuite) TestJSONRoundTrip(c *gc.C) {
+	fpkp, err := GenerateKey()
```

Comment:
If you were looking for other suggestions for this bikeshed, you could  make a MustGenerateKey that panics on err, that would remove some code in the various tests that generate keys